arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,139 @@
+
+![pie-slice](./assets/l2beat-pie-chart.png)
+
+With BOLD at its core, Arbitrum charts a course towards being recognized as a Stage 2 rollup by addressing the currently yellow (above) State Validation wedge in [L2Beat's risk analysis pie chart](https://l2beat.com/scaling/summary). BOLD contributes to a more decentralized, efficient, and robust rollup ecosystem. Additionally, BOLD will be available as an upgrade to all Orbit chains who wish to adopt it to reap the aforementioned benefits.
+
+### How is this possible?
+The BOLD protocol provides the guardrails and rules for how validators produce, validate, and challenge claims about the state of an Arbitrum chain. Since Arbitrum’s state is deterministic, there will always be only 1 correct state for a given input of on-chain operations and transactions. The beauty of BOLD’s design is centered around this very fact: that there is only 1 correct state on Arbitrum and therefore anyone defending that correct state will always prevail.
+
+Let’s dive in to an overview of how BOLD actually works.
+
+1. **An assertion is made:** Validators begin by taking the most recent confirmed L2 block, called `Block A`, and assert that some number of transactions afterwards, using Nitro’s deterministic State Transition Function (STF), will result in an end state, `Block Z`. If a validator claims that the end state represented by `Block Z` is correct, they will stake on `Block Z` and propose that state to be posted to Ethereum. If nobody disagrees after a certain amount of time, known as the challenge period, then the state represented by the L2 `Block Z` is confirmed as the correct state of an Arbitrum chain. However, if someone disagrees with the end state `Block Z`, they can submit a challenge. This is where BOLD comes in to play.
+2. **A challenge is opened:** When another validator observes and disagrees with the end state represented by `Block Z`, they can permissionlessly open a challenge by asserting and staking a claim on a different end state, represented by an L2 `Block Y`. At this point in time, there are now 2 asserted states: `Block A → Block Z` and `Block A → Block Y`. This pair of asserted states is called *an edge* while a Merkle tree of asserted states from some start to end point (e.g. `Block A → Block Z`) is more formally known as a *history commitment.* It is important to note that Ethereum at this point in time has no notion of which edges are correct or incorrect - edges are simply a portion of a claim about the history of the chain from some end state all the way back to some initial state.
```

Comment:
I would confirm the definition of "edge" given here. This is Bold specific and I'm not completely sure of the right definition.